go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/openshift/ingress-node-firewall
 
-go 1.21
+go 1.21.0
```

Comment:
I've seen this happen on a few go mod changes recently. If you have `toolchain` do you actually need to be so specific with `go 1.21.0` or would `go 1.21` suffice?

Reply:
`make vendor` is the one changing it for me